tools/WinObjC.Tools/vsimporter-templates/packageable.project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "WinObjC.Packaging": "*",
+    "WinObjC.Frameworks": "*",
```

Comment:
Yes.

---
In reply to: [100940695](https://github.com/Microsoft/WinObjC/pull/1958#discussion_r100940695) [](ancestors = 100940695,100940627)